tests/test_layers.py

```diff
@@ -1017,6 +1017,34 @@ def model_func(wrapped_import, inputs: torch.Tensor):
   verify_torch_and_convert_to_returnn(model_func, inputs=x)
 
 
+def test_movedim_2():
+  n_batch, n_time, n_feature = 3, 5, 7
+
+  def model_func(wrapped_import, inputs: torch.Tensor):
+    if typing.TYPE_CHECKING or not wrapped_import:
+      import torch
+    else:
+      torch = wrapped_import("torch")
+    out = torch.movedim(inputs, 0, 2)
+    return out
+
+  rnd = numpy.random.RandomState(42)
+  x = rnd.normal(0., 1., (n_time, n_batch, n_feature)).astype("float32")
```

Comment:
Still wrong order? Shouldn't it be:
```suggestion
  x = rnd.normal(0., 1., (n_batch, n_feature, n_time)).astype("float32")
```

Reply:
Yes, you're right, sorry.